src/Altinn.App.Core/Models/Layout/Components/GridComponent.cs

```diff
@@ -0,0 +1,64 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Altinn.App.Core.Models.Expressions;
+
+namespace Altinn.App.Core.Models.Layout.Components;
+
+/// <summary>
+/// Component specialisation for repeating groups with maxCount > 1
+/// </summary>
+public class GridComponent : GroupComponent
+{
+    /// <summary>
+    /// Constructor for RepeatingGroupComponent
+    /// </summary>
+    public GridComponent(string id, string type, IReadOnlyDictionary<string, string>? dataModelBindings, IEnumerable<BaseComponent> children, IEnumerable<string>? childIDs, Expression? hidden, Expression? required, Expression? readOnly, IReadOnlyDictionary<string, string>? additionalProperties) :
+        base(id, type, dataModelBindings, children, childIDs, hidden, required, readOnly, additionalProperties)
+    { }
+}
+
+
+/// <inheritdoc/>
```

Comment:
Fix documentation for this class. Class and method is using `/// <inheritdoc/>`, but the class does not extend any class so there is nothin to inherit.
Class is fairly simple so just short text are needed I think :)